index.bs

```diff
@@ -803,9 +763,26 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
               value in {{MediaCapabilitiesEncodingInfo/configuration}}.
             </li>
             <li>
-              If the user agent is able to encode the media represented by
-              <var>configuration</var>, set {{MediaCapabilitiesInfo/supported}}
-              to <code>true</code>. Otherwise set it to <code>false</code>.
+              If <code>configuration.video</code> [=map/exists=], let
+              |videoMimeSupport| be the result of running [$Check MIME Type
+              Support$] with <code>configuration.video.\[[mimeType]]</code>,
+              the |media| video, and |configuration|'s
+              {{MediaEncodingConfiguration/type}}, otherwise
+              let |videoMimeSupport| be <code>undefined</code>.
+            </li>
+            <li>
+              If <code>configuration.audio</code> [=map/exists=], let
+              |audioMimeSupport| be the result of running [$Check MIME Type
+              Support$] with <code>configuration.video.\[[mimeType]]</code>,
+              the |media| video, and |configuration|'s
+              {{MediaEncodingConfiguration/type}}, otherwise
+              let |audioMimeSupport| be <code>undefined</code>.
+            </li>
+            <li>
+              If either |videoMimeSupport| or |audioMimeSupport| is
+              <code>unsupported</code>, set {{MediaCapabilitiesInfo/supported}}
```

Comment:
If the verdict is `unsupported`, then logically the user agent cannot encode the media smoothly or efficiently.   So the steps to check that below now are redundant.